### PR TITLE
[147879] Add tests for :bypass_job_queue option of AsyncJob

### PR DESCRIPTION
**Pull request by s417-lama**
Tests for https://github.com/access-company/antikythera/pull/24

https://acsmine.tok.access-company.com/redmine/issues/147879

### CODE REVIEW

File: test/async_job_test.exs [hunk 1]
```diff
@@ -62,7 +62,10 @@ defmodule Testgear.AsyncJobTest do
   end
 
   defp register_job(todo, options \\ []) do
-    {:ok, _} = TestAsyncJob.register(%{todo: todo}, @epool_id, options)
+    case TestAsyncJob.register(%{todo: todo}, @epool_id, options) do
+      {:ok, _} -> :ok
+      error    -> error
+    end
```
Comment (skirino) [inline test/async_job_test.exs]: So far we've checked that `TestAsyncJob.register/3` succeeded by matching against `{:ok, _}`.
Now that it's not enforced, we have to check its return values.
Please change all existing calls of `register_job/1,2` as follows:

`register_job(:send)` => `assert register_job(:send) == :ok`

Comment (s417-lama) [reply test/async_job_test.exs]: Fixed in 9760a9011839565aca07c5f2e1e60a5e776024c4

**Comment by skirino** [comment]
LGTM
And thank you for correcting erlang/elixir versions during tests!

**Comment by ttokit** [comment]
LGTM!

**Comment by ymtszw** [comment]
# okay 🚀

**Pull request merged**